src/languages/en-US/events/guilds-logs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "logsEmbedTitle": "Guild Joined",
+    "logsEmbedDescription": "{{guild.name}} ({{guild.id}}) has joined the bot.",
+    "logsEmbedFieldsOwner": "Owner",
+    "logsEmbedFieldsMembers": "Members",
+    "logsEmbedFieldsCreated": "Created At",
+    "logsEmbedFieldsCreatedValue": "{{timestamp, dateTime}}",
+    "joinBotEmbedTitle": "Hello!",
+    "joinBotEmbedDescription": "Hello! I'm Birthdayy Bot, a bot that can help you celebrate birthdays! To get started, use this command for set configuration: {{command}}. If you need help, join our support server [here](https://join.birthdayy.xyz).",
```

Comment:
is this a link that works?

Reply:
It will have to be added later...

Reply:
or replace it with [birthdayy.xyz/discord](https://birthdayy.xyz/discord)?

Reply:
Yes, I would like not to use paths but subdomains directly, Is it possible ?

Reply:
Ah yes absolutly! We used to do it with the nginxproxymanager, we are not using this anymore tho, how would you suggest to do it now?

Reply:
I think it would be better to do it from cloudflare directly to reduce in addition the number of redirects in the process

Reply:
With Pagerules? Or is there a direct way in the DNS settings?

Reply:
With Redirect Rules